src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/notification/FailedPaymentApplicantNotificationHandler.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.unspec.handler.callback.notification;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.unspec.callback.Callback;
+import uk.gov.hmcts.reform.unspec.callback.CallbackHandler;
+import uk.gov.hmcts.reform.unspec.callback.CallbackParams;
+import uk.gov.hmcts.reform.unspec.callback.CaseEvent;
+import uk.gov.hmcts.reform.unspec.config.properties.notification.NotificationsProperties;
+import uk.gov.hmcts.reform.unspec.model.CaseData;
+import uk.gov.hmcts.reform.unspec.service.NotificationService;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.unspec.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.unspec.callback.CaseEvent.NOTIFY_APPLICANT_SOLICITOR1_FOR_FAILED_PAYMENT;
+
+@Service
+@RequiredArgsConstructor
+public class FailedPaymentApplicantNotificationHandler extends CallbackHandler implements NotificationData {
+
+    private static final List<CaseEvent> EVENTS = List.of(NOTIFY_APPLICANT_SOLICITOR1_FOR_FAILED_PAYMENT);
+    public static final String TASK_ID = "CreateClaimPaymentFailedNotifyApplicantSolicitor1";
+    private static final String REFERENCE_TEMPLATE = "failed-payment-applicant-notification-%s";
+
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return Map.of(
+            callbackKey(ABOUT_TO_SUBMIT), this::notifyClaimantSolicitorForFailedPayment
+        );
+    }
+
+    @Override
+    public String camundaActivityId() {
+        return TASK_ID;
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    private CallbackResponse notifyClaimantSolicitorForFailedPayment(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+
+        notificationService.sendMail(
+            notificationsProperties.getClaimantSolicitorEmail(),
+            notificationsProperties.getFailedPayment(),
+            addProperties(caseData),
+            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
+        );
+
+        return AboutToStartOrSubmitCallbackResponse.builder().build();
+    }
+
+    @Override
+    public Map<String, String> addProperties(CaseData caseData) {
+        return Map.of(
+            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+            SOLICITOR_REFERENCE, "claimant solicitor"
```

Comment:
is this the reference we capture duering claim creation journey? 
```suggestion
            SOLICITOR_REFERENCE, caseData.getSolicitorReferences().getApplicantSolicitor1Reference()
```

Reply:
it's just a placeholder notification now, the content will be updated in separate story